docs/how-to-guides/new-user-guides/authentication-permissions-and-global-configuration/authentication-config/configure-keycloak-oidc.md

```diff
@@ -35,6 +35,7 @@ If you have an existing configuration using the SAML protocol and want to switch
     `Name` | `Groups Mapper`
     `Mapper Type` | `Group Membership`
     `Token Claim Name` | `groups`
+    `Full group path` | `OFF`
```

Comment:
Are these additional fields added in any of the related keycloak tickets? I ask because the issue linked does not have the fields listed.

Reply:
This is just to align with an option available in keycloak. Currentl, you have to guess what is the correct option.